### PR TITLE
Remove using std::<foo> statements.

### PR DESCRIPTION
**Pull request by dj2**
Many of the files have using std::<foo> statements in them, but then the
use of <foo> will be inconsistently std::<foo> or <foo> scattered
through the file. This CL removes all of the using statements and
updates the code to have the required std:: prefix.

### CODE REVIEW

**Comment by dj2** [comment]
Thoughts? This seems clearer to me, but what do y'all think?

**Comment by dj2** [comment]
@alan-baker @s-perron thoughts on this cl?

File: test/val/val_layout_test.cpp [hunk 1]
```diff
@@ -136,53 +126,54 @@ INSTANTIATE_TEST_CASE_P(InstructionsOrder,
     // are not "All" in the below.
     //
     //                                  | Instruction                | Line(s) valid          | Lines to compile
```
Comment (s-perron) [inline test/val/val_layout_test.cpp]: The headers for this should should be realigned.

Comment (dj2) [reply test/val/val_layout_test.cpp]: done

**Comment by dj2** [comment]
Given @dneto0 is on the fence, @s-perron +1'd and @alan-baker is fine either way, I'm going to go ahead and merge this change.

**Pull request merged**